paddle/fluid/operators/shape_op.cc

```diff
@@ -58,6 +73,14 @@ Shape Operator.
 
 Return the shape of the input.
 )DOC");
+    AddAttr<bool>("use_mkldnn",
+                  "(bool, default false) Only used in mkldnn kernel")
+        .SetDefault(false);
+    AddAttr<std::string>(
+        "mkldnn_data_type",
+        "(string, default \"float32\"). Data type of mkldnn kernel")
+        .SetDefault("float32")
+        .InEnum({"float32", "bfloat16", "int8"});
```

Comment:
I wonder whether this attribute is necessary?  I do not see you use it.

Reply:
I think that it is needed for bf16 inference pass